css-three-column-layout/styles.css

```diff
@@ -0,0 +1,37 @@
+body {
+  background-color: rgb(71 148 212);
+}
+
+.image {
+  background-color: white;
+  width: 100px;
+  height: 100px;
+  border-radius: 100%;
+  padding: 18px;
+  margin: 0 auto;
+}
+
+section.text {
+  color: white;
+  text-align: center;
+  margin: 0 20%;
+}
+
+.box {
+  display: flex;
+  justify-content: space-around;
+}
+
+.col {
+  justify-content: center;
```

Comment:
`justify-content` has no effect if it's not being used alongside `display: flex`